chainer/initializers/__init__.py

```diff
@@ -37,7 +37,7 @@ def generate_array(initializer, shape, xp, dtype=None, device=None):
     Args:
         initializer: A callable object that takes :ref:`ndarray` and edits its
             value.
-        shape (tuple): Shape of a return array.
+        shape (int or tuple of int): Shape of a return array.
```

Comment:
Please separate this change, as it could be backported.

Reply:
Moved to #7450.